### PR TITLE
[Ready] Adds examine descs, air alarm construction descs

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
spellcheck: Added missing descriptions for objects that have alt+click functionality.
/:cl:

A bunch of items were missing examine messages about their alt+click functionality, I added them. Also, construction step descs for air alarms.

### CODE REVIEW

File: code/game/machinery/porta_turret/portable_turret.dm [hunk 1]
```diff
@@ -789,6 +789,12 @@
 		turrets |= T
 		T.cp = src
 
+/obj/machinery/turretid/examine(mob/user)
+	..()
+	if(issilicon(user) && (!stat & BROKEN))
+		to_chat(user, "<span class='notice'>Ctrl-Click [src] to [ enabled ? "disable" : "ensable"] turrets.</span>")
```
Comment (SpaceManiac) [inline code/game/machinery/porta_turret/portable_turret.dm]: "ensable" -> "enable"

File: code/game/machinery/doors/airlock.dm [hunk 2]
```diff
@@ -664,6 +664,12 @@
 		else
 			to_chat(user, "It looks very robust.")
 
+	if(issilicon(user) && (!stat & BROKEN))
+		to_chat(user, "<span class='notice'>Shift-Click [src] to [ density ? "open" : "close"] it.</span>")
+		to_chat(user, "<span class='notice'>Ctrl-Click [src] to [ locked ? "raise" : "drop"] its bolts.</span>")
+		to_chat(user, "<span class='notice'>Alt-Click [src] to [ secondsElectrified ? "un-electrify" : "permanently electrify"] it.</span>")
+		to_chat(user, "<span class='notice'>Ctrl+Shift-Click [src] to [ emergency ? "disable" : "enable"] emergency access.</span>")
```
Comment (SpaceManiac) [inline code/game/machinery/doors/airlock.dm]: Should be consistent about whether `+` or `-` is used to combine modifiers, ditto skittish

File: code/modules/vehicles/ridden.dm [hunk 3]
```diff
@@ -14,7 +14,12 @@
 /obj/vehicle/ridden/examine(mob/user)
 	. = ..()
 	if(key_type)
-		to_chat(user, "<span class='notice'>Put a key inside it by clicking it with the key. If there's a key inside, you can remove it via Alt-Click!</span>")
+		if(!inserted_key)
+			to_chat(user, "<span class='notice'>Put a key inside it by clicking it with the key.</span>")
+		else
+			to_chat(user, "<span class='notice'>Alt-Click [src] to remove the key.</span>")
+	else
+		return
```
Comment (SpaceManiac) [inline code/modules/vehicles/ridden.dm]: redundant `else`...`return`

**Review by SpaceManiac — CHANGES_REQUESTED** [review]

File: code/game/machinery/doors/airlock.dm (hunk 2 above)
Comment (81Denton) [reply code/game/machinery/doors/airlock.dm]: I think I'll stick with `-`

**Comment by epochayur** [comment]
based

**Review by SpaceManiac — APPROVED** [review]
lgtm

**Pull request merged**